rising/transforms/compose.py

```diff
@@ -42,7 +68,12 @@ def __init__(self, *transforms,
         super().__init__(grad=True)
         if isinstance(transforms[0], Sequence):
             transforms = transforms[0]
-        self.transforms = transforms
+
+        for idx, trafo in enumerate(transforms):
+            if not isinstance(trafo, torch.nn.Module):
+                transforms[idx] = _TransformWrapper(trafo)
+
+        self.transforms = torch.nn.ModuleList(transforms)
```

Comment:
[Optional] Should we do this with a property? (with a setter)

Reply:
I don't know, if this wouldn't be an overkill for this one time, we assign it, but I can definitely add a check, whether this already is a module list (but even if it is, this should be fine as calls would be propagated through)

Reply:
Yes :) It was just optional ^^